### DISCUSSION TITLE
Catching errors

### DISCUSSION THREAD

**Discussion by bebejane** [vercel/satori#317] [Q&A]
How does one catch errors when creating new ImageResponse in vercel edge api function?

Errors are printed in console but when wrapping the call with try/catch no error is thrown.

**Comment by iamhectorsosa** [comment]
@bebejane can you share your code or source? When using a try catch block, how are you returning the error? This example: [Dynamic text generated as image](https://vercel.com/docs/concepts/functions/edge-functions/og-image-examples#dynamic-text-generated-as-image) works as intended for error handling the request. Additionally, what are you looking to do with the error?
